solr/bin/solr

```diff
@@ -2153,16 +2160,17 @@ function start_solr() {
   SOLR_JETTY_ADDL_CONFIG="$3"
 
   # define default GC_TUNE
-  if [ -z ${GC_TUNE+x} ]; then # I think this check is backwards?
-      GC_TUNE=('-XX:+UseG1GC' \
+  if [ -z "${GC_TUNE}" ]; then
+      GC_TUNE_ARR=('-XX:+UseG1GC' \
```

Comment:
@madrob You added that comment about backward check. I agree, cause the `+x` thing  will always pass the condition, so currently it will never be possibel to override `GC_TUNE` for the user, agree? I'm pretty sure that my fix here is enough

Reply:
I think this will fail if GC_TUNE is unset now? Might not fail in regular operation, but in BATS it might be more strict and complain.

Reply:
I tested this
```bash
$ unset GC_TUNE  
$ if [ -z "${GC_TUNE}" ]; then                                                                                              1 ↵  10130  13:11:57
  echo "DEFAULTS"
else
  echo "OVERRIDE"
fi
DEFAULTS

$ GC_TUNE=foo
$ if [ -z "${GC_TUNE}" ]; then                                                                                                ✔  10128  13:11:28
  echo "DEFAULTS"
else
  echo "OVERRIDE"
fi
OVERRIDE
```